hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileUpdateTest.java

```diff
@@ -407,34 +411,46 @@ void appliesNewContentUpgradeFile() {
     void appliesNewExpiryViaMeta() {
         refreshStoresWithCurrentFileInBothReadableAndWritable();
 
-        final var expiry = Timestamp.newBuilder().seconds(1_234_568L).build();
+        final var expiry = Timestamp.newBuilder().seconds(3_456_789L).build();
         final var op = OP_BUILDER.fileID(wellKnownId()).expirationTime(expiry).build();
-        final var txBody = TransactionBody.newBuilder().fileUpdate(op).build();
+        final var txBody = TransactionBody.newBuilder()
+                .fileUpdate(op)
+                .transactionID(TransactionID.newBuilder()
+                        .transactionValidStart(
+                                Timestamp.newBuilder().seconds(100L).build())
+                        .build())
+                .build();
         when(handleContext.body()).thenReturn(txBody);
         given(handleContext.verificationFor(Mockito.any(Key.class))).willReturn(signatureVerification);
         given(signatureVerification.failed()).willReturn(false);
 
         subject.handle(handleContext);
 
         final var newFile = writableFileState.get(fileId);
-        assertEquals(1_234_568L, newFile.expirationSecond());
+        assertEquals(3_456_789L, newFile.expirationSecond());
     }
 
     @Test
     void appliesNewExpiryLowerExpirationTimeViaMeta() {
         refreshStoresWithCurrentFileInBothReadableAndWritable();
 
-        final var expiry = Timestamp.newBuilder().seconds(1_234_566L).build();
+        final var expiry = Timestamp.newBuilder().seconds(3_456_789L).build();
         final var op = OP_BUILDER.fileID(wellKnownId()).expirationTime(expiry).build();
-        final var txBody = TransactionBody.newBuilder().fileUpdate(op).build();
+        final var txBody = TransactionBody.newBuilder()
+                .fileUpdate(op)
+                .transactionID(TransactionID.newBuilder()
+                        .transactionValidStart(
+                                Timestamp.newBuilder().seconds(100L).build())
+                        .build())
+                .build();
         when(handleContext.body()).thenReturn(txBody);
         given(handleContext.verificationFor(Mockito.any(Key.class))).willReturn(signatureVerification);
         given(signatureVerification.failed()).willReturn(false);
 
         subject.handle(handleContext);
 
         final var newFile = writableFileState.get(fileId);
-        assertEquals(1_234_567L, newFile.expirationSecond());
+        assertEquals(3_456_789L, newFile.expirationSecond());
```

Comment:
This number appears a few times in the file.  Might be worth introducing a constant for it.

Reply:
done thanks